examples/BLE_reader/BLE_reader.ino

```diff
@@ -24,6 +24,6 @@ void setup() {
 
 void loop() {
   bead.clear();
-  bead.setBloch_deg_smooth(bead.t_ble, bead.p_ble, bead.c_ble);
```

Comment:
let's split this in two:
```
state.setTheta(bead.ble_theta);
state.setPhi(bead.ble_phi);
bead.setBlochState(state);
```

Think of `bead` as an interface to the hardware (imu, leds, ble) and all logic related to quantum information science is in your state object, not in the bead object.